messages/teleporter/message_manager.go

```diff
@@ -105,6 +108,19 @@ func (m *messageManager) ShouldSendMessage(warpMessageInfo *vmtypes.WarpMessageI
 	if !ok {
 		return false, fmt.Errorf("relayer not configured to deliver to destination. destinationChainID=%s", destinationChainID.String())
 	}
+
+	// If allowedDestinations is empty, then all destinations are allowed
+	// If allowedDestinations is not empty, then only the allowed destinations are allowed
+	if _, exist := m.allowedDestinations[destinationChainID]; !exist && len(m.allowedDestinations) > 0 {
+		m.logger.Info(
+			"Destination chain not allowed to receive messages.",
```

Comment:
I would rephrase this as "Relayer not configured to relay between source and destination", and add the `sourceChainID` to the log.